.travis.yml

```diff
@@ -18,6 +18,7 @@ before_install:
   # extralibs
   - wget https://github.com/ericsun2/sandbox/raw/master/extralibs/extralibs.zip
   - mkdir -p metadata-etl/extralibs; unzip extralibs.zip -d metadata-etl/extralibs
+  - mkdir -p RestLiClient/extralibs; unzip extralibs.zip -d RestLiClient/extralibs
```

Comment:
yes, discussed with Eric earlier today, he suggested we put all of them in one sandbox.  The other option is to put one in root folder (but may need split the project in the future).